src/prob/pf.jl

```diff
@@ -269,7 +269,7 @@ function PowerFlowData(data_math::Dict{String,<:Any}, v_start::Dict{<:Any,<:Any}
     for (comp_type, comp_interface) in _CPF_COMPONENT_INTERFACES
         for (id, comp) in data_math[comp_type]
             if comp[comp_status_prop[comp_type]]==1
-                (bts, nr_vns, y_prim, cc) = comp_interface(comp, v_start)
+                (bts, nr_vns, y_prim, cc, ccd) = comp_interface(comp, v_start)
```

Comment:
Can you put a comment somewhere appropriate what the semantics of the variables are:
- bts - bus-terminals for the component
- nr_vns - number of virtual nodes (?)
- y_prim primitive admittance matrix for the component
- cc compensation current function handle for the component
- ccd total current function handle for the component (?) (what does the 'd' stand for?)